tests/cupyx_tests/scipy_tests/signal_tests/test_spectral.py

```diff
@@ -964,3 +964,268 @@ def test_check_NOLA_fail(self, xp, scp):
         for setting in settings_fail:
             result.append(scp.signal.check_NOLA(*setting))
         return result
+
+
+@testing.with_requires('scipy>1.8.0')
```

Comment:
Seems that the CI has scipy==1.8.1 or newer, so the test was not skipped.
```suggestion
@testing.with_requires('scipy>=1.9.0')
```